### PR TITLE
Typography margin

### PR DESCRIPTION
**Pull request by mapmeld**
Add a rule removing margin below Antd Typography - this will be used to complete https://github.com/determined-ai/determined/pull/8513

### CODE REVIEW

File: src/kit/Typography.module.scss [hunk 1]
```diff
@@ -87,3 +87,6 @@
   font-size: var(--body-size-default);
   line-height: var(--body-ln-default);
 }
+:global(.ant-typography) {
+  margin-bottom: 0;
```
Comment (johnkim-det) [inline src/kit/Typography.module.scss:91]: also `margin-top`?

Comment (mapmeld) [reply src/kit/Typography.module.scss:91]: ✔️

**Comment by johnkim-det** [comment]
checking the examples, looks like it's not always getting overwritten

**Comment by mapmeld** [comment]
Updated the css rule for precedence

**Pull request merged**